### PR TITLE
Add link to interview on each screen

### PR DESCRIPTION
**Pull request by nonprofittechy**

### CODE REVIEW

File: docassemble/MAVirtualCourt/data/questions/basic-questions.yml [hunk 1]
```diff
@@ -29,7 +33,7 @@ objects:
   - plaintiffs: PeopleList
   - plaintiff: Individual
   - other_parties: PeopleList
-  - courts: DAList.using(object_type=MACourt, there_are_any=True, there_is_another=False) # Normally let's only have one court...
+  - courts: DAList.using(auto_gather=False, gathered=True, complete_attribute='name') # Normally let's only have one court...
```
Comment (plocket) [inline docassemble/MAVirtualCourt/data/questions/basic-questions.yml:36]: Is this change necessary for this PR?

**Comment by plocket** [comment]
From what I can see, the expected interaction is that someone would right-click on the link and copy the link location, because clicking on the link is very confusing - it just takes you to the page you're on. Is there a way we can make that less confusing? I'm not sure how many people are really familiar with that behavior.

File: docassemble/MAVirtualCourt/data/questions/test.yml [hunk 2]
```diff
@@ -1,54 +1,6 @@
-metadata:
-  title: Default playground interview
-  short title: Test
-  comment: This is a learning tool.  Feel free to write over it.
```
Comment (plocket) [inline docassemble/MAVirtualCourt/data/questions/test.yml:4]: This file was sort of gutted. Is this change needed for this PR?

**Comment by plocket** [comment]
> Is there a way we can make that less confusing?

Other options:
1. A link that sends an email to one of us.
1. A link to a google form they can turn in with the url pre-populated for them
1. A 'copy this' type of interaction (where clicking on the link copies the link to their clipboard - no, I don't know how to do this yet) with instructions to share the link in Slack, with a certain email address, on a google form, whatever.

File: docassemble/MAVirtualCourt/data/questions/test.yml (hunk 2 above)
Comment (nonprofittechy) [reply docassemble/MAVirtualCourt/data/questions/test.yml:4]: It shouldn't be in the repo--it's a scratch pad. But it's not relied on by anything else.

File: docassemble/MAVirtualCourt/data/questions/basic-questions.yml (hunk 1 above)
Comment (nonprofittechy) [reply docassemble/MAVirtualCourt/data/questions/basic-questions.yml:36]: > 
> 
> > Is there a way we can make that less confusing?
> 
> Other options:
> 
>     1. A link that sends an email to one of us.
> 
>     2. A link to a google form they can turn in with the url pre-populated for them
> 
>     3. A 'copy this' type of interaction (where clicking on the link copies the link to their clipboard - no, I don't know how to do this yet) with instructions to share the link in Slack, with a certain email address, on a google form, whatever.

Grea

> Other options:
> 
>     1. A link that sends an email to one of us.

These are great ideas, but outside of scope of this PR. I think this is just a stopgap until someone creates the form/process described above, which I think is really ideal. I'd prefer that someone not to be me.

Comment (plocket) [reply docassemble/MAVirtualCourt/data/questions/basic-questions.yml:36]: Sure, I hear that. I'll make a commit on this branch to add instructions at least.

**Comment by plocket** [comment]
As far as the other two changes go, we're deciding to test them and go with them for now, aiming for more self-contained PRs moving forward.
